README.md

```diff
@@ -1,10 +1,10 @@
 # Theia
 
-Theia will contain Network Flow Visibility functions which are split from the
-main repo. Flow exporter and flow aggregator will be kept in [Antrea](https://github.com/antrea-io/antrea)
-repo, other modules, such as Clickhouse and Grafana related parts, will be moved
-to this new repo.
+Theia contains Network Flow Visibility functions which are exacted from the
+[Antrea main repo](https://github.com/antrea-io/antrea). While flow exporter and
+flow aggregator are kept in the Antrea repo, other flow visibility modules, such
+as Clickhouse and Grafana related ones, will be moved to this new repo.
```

Comment:
not introduced in this PR

Clickhouse -> ClickHouse

Reply:
Fixed.